samples/MASMessagingSample/src/main/AndroidManifest.xml

```diff
@@ -1,49 +1,72 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
   ~ Copyright (c) 2016 CA. All rights reserved.
   ~
   ~ This software may be modified and distributed under the terms
   ~ of the MIT license.  See the LICENSE file for details.
   ~
-  -->
+-->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.ca.mas.masmessagingsample">
+    package="com.ca.mas.masmessagingsample">
+    <uses-feature
+        android:name="android.hardware.nfc"
+        android:required="true" />
+    <uses-permission android:name="android.permission.NFC" />
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
 
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity
+        <!--<activity
```

Comment:
If this activity's been completely replaced, then we can just delete the commented section.